views/github-configuration.hbs

```diff
@@ -28,98 +28,100 @@
+            </div>
 
-          <div class="githubOrgs__table">
-            {{#each installations}}
-              <div class='Box-row clearfix '>
-                {{!-- Organizations --}}
-                <div class='getConfiguration__orgContent org__cell'>
-                  <img
-                    class='avatar float-left mr-2'
-                    src='{{account.avatar_url}}'
-                    alt='Github organization avatar'
-                    height='25px'
-                    width='25px'
-                  />
-                  <span class='text-bold text-gray-dark'><a
-                      href='{{html_url}}'
-                    >{{account.login}}</a></span>
-                </div>
-
-                {{!-- Repository access --}}
-                <div class='getConfiguration__orgContent org__cell'>
-                  <p class="getConfiguration__orgContent__repoSelection">{{repoAccessType repository_selection}}</p>
-                  <p class="getConfiguration__orgContent__numberOfRepos">{{numberOfRepos}}</p>
-                  {{#if admin}}
-                    <a
-                      href='/github/subscriptions/{{id}}'
-                    >
+            <div class="githubOrgs__table">
+              {{#if installations}}
+                {{#each installations}}
+                  <div class='githubOrgs__table__row clearfix'>
+                    {{!-- Organizations --}}
+                    <div class='getConfiguration__orgContent org__cell'>
                       <img
-                        class="getConfiguration__orgContent_edit"
-                        src='/public/assets/edit-icon.png'
-                        alt='Editing icon'
+                        class='avatar float-left mr-2'
+                        src='{{account.avatar_url}}'
+                        alt='Github organization avatar'
                         height='25px'
                         width='25px'
                       />
-                    </a>
-                  {{/if}}
-                </div>
+                      <span class='text-bold text-gray-dark'><a
+                          href='{{html_url}}'
+                        >{{account.login}}</a></span>
+                    </div>
+
+                    {{!-- Repository access --}}
+                    <div class='getConfiguration__orgContent org__cell'>
+                      <p class="getConfiguration__orgContent__repoSelection">{{repoAccessType repository_selection}}</p>
+                      <p class="getConfiguration__orgContent__numberOfRepos">{{numberOfRepos}}</p>
+                      {{#if admin}}
+                        <a
+                          href='/github/subscriptions/{{id}}'
+                        >
+                          <img
+                            class="getConfiguration__orgContent_edit"
+                            src='/public/assets/edit-icon.png'
+                            alt='Editing icon'
```

Comment:
I'd probably just make this `'Edit'` as it's tied to the action and not a standalone image.